run-tests.sh

```diff
@@ -1,73 +1,21 @@
 #!/usr/bin/env bash
 
-
 set -eu
 
-declare status_code
+remove_quotes () {
+  echo "$1" | tr -d '"'
+}
+
+declare error_code
 # shellcheck disable=SC2154
-# status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "$CFN_HelloWorldApi")"
-status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200")"
+#The CFN variables seem to include quotes when used in tests these must be removed before assigning them.
+CFN_CICBackendURL_NoQuotes=$(remove_quotes $CFN_CICBackendURL)
```

Comment:
```suggestion
CFN_CICBackendURL_NoQuotes=$(remove_quotes "$CFN_CICBackendURL")
```
my linter is complaining about this!